### PR TITLE
Add contributing md

### PR DESCRIPTION
**Pull request by iskandr**
[Review on Reviewable](https://reviewable.io/reviews/hammerlab/varcode/41)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+-----------------
+* Varcode is written in Python and adheres to the [PEP8](https://www.python.org/dev/peps/pep-0008/)
+style guidelines.
+* Please ensure that your submissions work under both Python 2.7.x and Python >= 3.3.
+* New features should start with a GitHub issue explaining their scope and rationale.
+* New code should be accompanied by comprehensive unit tests.
+
+Licensing
+---------
+Varcode is licensed under the Apache 2.0 license. Your code is assumed to be as well.
```
Comment (tavinathanson) [inline CONTRIBUTING.md]: How about info on filing, PRs, etc. ala https://github.com/hammerlab/guacamole/blob/master/CONTRIBUTING.md?

**Comment by tavinathanson** [comment]
Is this PR supposed to have code changes too?

**Comment by iskandr** [comment]
The code changes are because this branch is off another than Tim is reviewing, I'll rebase it once that other branch is done.

**Comment by tavinathanson** [comment]
:+1:

**Comment by tavinathanson** [comment]
Merge away when ready

**Pull request merged**